static/i18n/en/app.json

```diff
@@ -1148,6 +1148,7 @@
     "title": "Current address",
     "for": "Address for <1><0>{{name}}</0></1>",
     "messageIfUnverified": "Verify that the shared address exactly matches the one on your device",
+    "messageIfVirtual": "Your {{name}} address cannot be confirmed on your Ledger device. Use at your own risk.",
```

Comment:
better scope this under a hedera key too

Reply:
👍🏻